serialization-jackson/src/main/kotlin/io/datalbry/precise/serialization/jackson/deserializer/SchemaDeserializer.kt

```diff
@@ -50,7 +50,7 @@ class SchemaDeserializer: StdDeserializer<Schema>(Schema::class.java) {
 
     private fun toEnumType(node: JsonNode): Type {
         val name = node.get("name").asText()
-        val values = node.findValuesAsText("values").toSet()
+        val values = node.get("values").mapValues { it.asText() }
```

Comment:
is the change necessariy?

Reply:
In this case `node.findValuesAsText("values")` returns a list with an empty string. We expect there a collection of strings that match the enum definition.

The change here `node.get("values")` returns a list of the expected values wrapped in a JsonNode.